prepare-efi.sh

```diff
@@ -28,17 +28,17 @@ fi
 # ``Error converting to codepage 850 Invalid argument``
 # ``Cannot initialize '::'``
 # This is due to the conversion table missing codepage 850, included in glibc-gconv-extra
-dnf install -y grub2 shim dosfstools mtools glibc-gconv-extra $PACKAGES
+dnf install -y grub2 shim dosfstools mtools glibc-gconv-extra "${PACKAGES}"
```

Comment:
maybe add disable=SC2086 here?
the PACKAGES list will be considered as a single string, afraid this will break the build

Reply:
PACKAGES is defined above to be a single word `grub2-efi-x86`, so it will not break. Variable naming is misleading.